app/Eloquent/Post.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Post extends Model
+{
+    protected $fillable = [
+        'title',
+        'content',
+        'priority',
+        'public',    
+    ];
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Eloquent/Post.php</br>Whitespace found at end of line